File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/addressspace/nodes/builtin/UaObjects.java
package org.opcfoundation.webserver.addressspace.nodes.builtin;

import org.eclipse.milo.opcua.stack.core.NodeIds;
import org.eclipse.milo.opcua.stack.core.types.builtin.LocalizedText;
import org.opcfoundation.webserver.addressspace.nodes.UaObject;

public class UaObjects {
    public final static UaObject RootFolder = new UaObject(NodeIds.RootFolder, "Root", new LocalizedText("Root"), UaObjectTypes.FolderType);
    public final static UaObject ObjectsFolder = new UaObject(NodeIds.ObjectsFolder, "Objects", new LocalizedText("Objects"), UaObjectTypes.FolderType);
    public final static UaObject Server = new UaObject(NodeIds.Server, "Server", new LocalizedText("Server"), UaObjectTypes.ServerType);

    public final static UaObject TypesFolder = new UaObject(NodeIds.TypesFolder, "Types", new LocalizedText("Types"), UaObjectTypes.FolderType);
    public final static UaObject ObjectTypesFolder = new UaObject(NodeIds.ObjectTypesFolder, "ObjectTypes", new LocalizedText("ObjectTypes"), UaObjectTypes.FolderType);
    public final static UaObject VariableTypesFolder = new UaObject(NodeIds.VariableTypesFolder, "VariableTypes", new LocalizedText("VariableTypes"), UaObjectTypes.FolderType);
    public final static UaObject EventTypesFolder = new UaObject(NodeIds.EventTypesFolder, "EventTypes", new LocalizedText("EventTypes"), UaObjectTypes.FolderType);
    public final static UaObject DataTypesFolder = new UaObject(NodeIds.DataTypesFolder, "DataTypes", new LocalizedText("DataTypes"), UaObjectTypes.FolderType);
    public final static UaObject ReferenceTypesFolder = new UaObject(NodeIds.ReferenceTypesFolder, "ReferenceTypes", new LocalizedText("ReferenceTypes"), UaObjectTypes.FolderType);
    public final static UaObject ViewsFolder = new UaObject(NodeIds.ViewsFolder, "Views", new LocalizedText("Views"), UaObjectTypes.FolderType);

    public final static UaObject ModellingRule_Mandatory = new UaObject(NodeIds.ModellingRule_Mandatory, "Mandatory", new LocalizedText("Mandatory"),UaObjectTypes.ModellingRuleType);
    public final static UaObject ModellingRule_Optional = new UaObject(NodeIds.ModellingRule_Optional, "Optional", new LocalizedText("Optional"),UaObjectTypes.ModellingRuleType);
    public final static UaObject ModellingRule_MandatoryPlaceHolder = new UaObject(NodeIds.ModellingRule_MandatoryPlaceholder, "MandatoryPlaceholder", new LocalizedText("MandatoryPlaceholder"),UaObjectTypes.ModellingRuleType);
    public final static UaObject ModellingRule_OptionalPlaceHolder = new UaObject(NodeIds.ModellingRule_OptionalPlaceholder, "OptionalPlaceholder", new LocalizedText("OptionalPlaceholder"),UaObjectTypes.ModellingRuleType);
}

File: opcua-webservice-node/source/src/server/addressspace/nodes/builtin/UaObjects.ts
import { ObjectIds, UaLocalizedText, UaNodeId } from "opcua-webapi-ts";
import { UaObject } from "../UaObject";
import { UaObjectTypes } from "./UaObjectTypes";

export class UaObjects {
    static RootFolder = new UaObject(UaNodeId.from(ObjectIds.RootFolder), "Root", new UaLocalizedText("Root"), UaObjectTypes.FolderType);
    static ObjectsFolder = new UaObject(UaNodeId.from(ObjectIds.ObjectsFolder), "Objects", new UaLocalizedText("Objects"), UaObjectTypes.FolderType);
    static Server = new UaObject(UaNodeId.from(ObjectIds.Server), "Server", new UaLocalizedText("Server"), UaObjectTypes.ServerType);

    static TypesFolder = new UaObject(UaNodeId.from(ObjectIds.TypesFolder), "Types", new UaLocalizedText("Types"), UaObjectTypes.FolderType);
    static ObjectTypesFolder = new UaObject(UaNodeId.from(ObjectIds.ObjectTypesFolder), "ObjectTypes", new UaLocalizedText("ObjectTypes"), UaObjectTypes.FolderType);
    static VariableTypesFolder = new UaObject(UaNodeId.from(ObjectIds.VariableTypesFolder), "VariableTypes", new UaLocalizedText("VariableTypes"), UaObjectTypes.FolderType);
    static EventTypesFolder = new UaObject(UaNodeId.from(ObjectIds.EventTypesFolder), "EventTypes", new UaLocalizedText("EventTypes"), UaObjectTypes.FolderType);
    static DataTypesFolder = new UaObject(UaNodeId.from(ObjectIds.DataTypesFolder), "DataTypes", new UaLocalizedText("DataTypes"), UaObjectTypes.FolderType);
    static ReferenceTypesFolder = new UaObject(UaNodeId.from(ObjectIds.ReferenceTypesFolder), "ReferenceTypes", new UaLocalizedText("ReferenceTypes"), UaObjectTypes.FolderType);
    static ViewsFolder = new UaObject(UaNodeId.from(ObjectIds.ViewsFolder), "Views", new UaLocalizedText("Views"), UaObjectTypes.FolderType);

    static ModellingRule_Mandatory = new UaObject(UaNodeId.from(ObjectIds.ModellingRule_Mandatory), "Mandatory", new UaLocalizedText("Mandatory"), UaObjectTypes.ModellingRuleType);
    static ModellingRule_Optional = new UaObject(UaNodeId.from(ObjectIds.ModellingRule_Optional), "Optional", new UaLocalizedText("Optional"), UaObjectTypes.ModellingRuleType);
    static ModellingRule_MandatoryPlaceHolder = new UaObject(UaNodeId.from(ObjectIds.ModellingRule_MandatoryPlaceholder), "MandatoryPlaceholder", new UaLocalizedText("MandatoryPlaceholder"), UaObjectTypes.ModellingRuleType);
    static ModellingRule_OptionalPlaceHolder = new UaObject(UaNodeId.from(ObjectIds.ModellingRule_OptionalPlaceholder), "OptionalPlaceholder", new UaLocalizedText("OptionalPlaceholder"), UaObjectTypes.ModellingRuleType);
}